main/src/sphexa/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ if(CMAKE_CUDA_COMPILER)
     target_compile_definitions(${exename}-cuda PRIVATE USE_CUDA)
     target_link_libraries(${exename}-cuda PRIVATE ryoanji sph_gpu io sim_init_gpu propagator_gpu observables_gpu OpenMP::OpenMP_CXX
         ${MPI_CXX_LIBRARIES} CUDA::cudart)
+    target_link_libraries(${exename}-cuda PRIVATE disk_gpu)
```

Comment:
`disk_gpu` should be linked to `propagator_gpu`

Reply:
changed.